cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,28 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        it('should have a visible play button and duration', () => {
+          cy.window().then(win => {
+            const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+            if (media) {
+              const {
+                durationISO8601,
+              } = media.model.blocks[1].model.blocks[0].model.versions[0];
+
+              cy.get('div[class^="StyledVideoContainer"]').within(() => {
```

Comment:
As discussed we should change this (later) to use a more reliable selector - there is a [core pod issue for this](
https://github.com/bbc/simorgh/issues/3714) already